core/lib/logger.ts

```diff
@@ -65,12 +74,12 @@ class Logger
    * @param tag short tag specifying area where message was spawned (eg. cart, sync, module)
    * @param context meaningful data related to this message
    */
-  debug (message: string, tag: string = null, context: any = null) : () => void {
+  debug (message: any, tag: string = null, context: any = null) : () => void {
```

Comment:
There are plenty of occurrences passing the objects instead of a message for debugging principles. [Just some example](https://github.com/pkarw/vue-storefront/blob/672ff09677956104ffb29620fe0f7ac997190d23/core/modules/user/components/Register.ts#L29)